src/app/mobile-layout/mobile-time/mobile-time.component.html

```diff
@@ -0,0 +1,130 @@
+              <td>
+                <mat-icon tooltip="Remove this activity from the schedule." (click)="removeSpiritActivity()" class="iconButton" aria-label="Remove Activity" aria-hidden="false">clear</mat-icon>
+              </td>
+            </tr>
+          }
+        </table>
+      }
+
+    <table>
+      @for (entry of activityService.activityLoop; track entry; let indexOfelement = $index) {
+        @if (activityService.getActivityName(entry.activity) !== '') {
+          <tr id="activityLoopIndex{{indexOfelement}}">
+            <td [ngClass]="{'currentRow': indexOfelement === activityService.currentIndex, 'disabled': entry.disabled || entry.userDisabled || battleService.enemies.length > 0}" id="activityLoopIndex0{{indexOfelement}}">
+              <mat-icon (click)="showActivity($event, entry.activity)"
+                tooltip="What is this activity?<br>Click to learn all about it!"
+                aria-label="Help" aria-hidden="false">
+                help
+              </mat-icon>
+              {{activityService.getActivityName(entry.activity)}} ({{entry.repeatTimes}} {{(entry.repeatTimes === 1) ? "day" : "days"}})
+            </td>
+            <td [ngClass]="{'currentRow': indexOfelement === activityService.currentIndex, 'disabled': entry.disabled || entry.userDisabled || battleService.enemies.length > 0}" id="activityLoopIndex1{{indexOfelement}}">
+              <div class="progress-bar">
+                <span
+                  [style.width.%]="100 * (indexOfelement !== activityService.currentIndex || entry.disabled || entry.userDisabled ? entry.repeatTimes : activityService.currentTickCount) / entry.repeatTimes">
+                </span>
+              </div>
+            </td>
+            <td [ngClass]="{'currentRow': indexOfelement === activityService.currentIndex, 'disabled': entry.disabled || entry.userDisabled || battleService.enemies.length > 0}" id="activityLoopIndex2{{indexOfelement}}">
+              <mat-icon tooltip="Spend fewer days on this.<br>Shift- or Ctrl-click to remove 10x<br>Shift-Ctrl-click to remove 100x"
+                (click)="onMinusClick(entry, $event)" class="iconButton" aria-label="Remove Days" aria-hidden="false">remove</mat-icon>
+              <mat-icon tooltip="Spend more days on this.<br>Shift- or Ctrl-click to add 10x<br>Shift-Ctrl-click to add 100x"
+                (click)="onPlusClick(entry, $event)" class="iconButton" aria-label="Add Days" aria-hidden="false">add</mat-icon>
+              <mat-icon tooltip="Disable this activity on the schedule." (click)="onDisableClick(entry)" class="iconButton" aria-label="Disable Activity" aria-hidden="false">cancel</mat-icon>
+              <mat-icon tooltip="Remove this activity from the schedule." (click)="onRemoveClick(entry)" class="iconButton" aria-label="Remove Activity" aria-hidden="false">clear</mat-icon>
+            </td>
+          </tr>
+        }
+      }
+    </table>
+  </div>
+}
+</div>
+<div class="btn-row">
```

Comment:
Would it make more sense to have the time control buttons broken out as their own component so that they can be a header or footer for everything?